### PR TITLE
[release-v7.13.x-blue][KIECLOUD][RHPAM-4609] Setting up Cachito tool.

### PR DESCRIPTION
**Pull request by bacciotti**
**Related Pull Requests**
* https://github.com/kiegroup/kie-cloud-operator/pull/853
* https://github.com/kiegroup/kie-cloud-operator/pull/854

### CODE REVIEW

File: image-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+- schema_version: 1
```
Comment (rgdoliveira) [inline image-prod.yaml:1]: You will need to update most of the entries in this file to match the ones from [BAMOE image.yaml](https://github.com/kiegroup/kie-cloud-operator/blob/release-v7.13.x-blue/image.yaml). Please check the file I pointed out and update this one based on it.

File: modules/builder-prod/module.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+schema_version: 1
+
+name: rhpam.builder.prod
```
Comment (rgdoliveira) [inline modules/builder-prod/module.yaml:3]: rename this module name to bamoe.builder.prod?

**Review by rgdoliveira — CHANGES_REQUESTED** [review]

File: image-prod.yaml (hunk 1 above)
Comment (bacciotti) [reply image-prod.yaml:1]: Good point. Changed!

**Pull request merged**